pkgs/development/tools/semantic-release/default.nix

```diff
@@ -0,0 +1,41 @@
+{ cctools
+, buildNpmPackage
+, fetchFromGitHub
+, lib
+, python3
+, stdenv
+}:
+
+buildNpmPackage rec {
+  pname = "semantic-release";
+  version = "21.0.5";
+
+  src = fetchFromGitHub {
+    owner = "semantic-release";
+    repo = "semantic-release";
+    rev = "v${version}";
+    hash = "sha256-QFBYIoOd1htSNj3xKl0HvmD1T068Jv+L/EqCbMrn0lw=";
+  };
+
+  npmDepsHash = "sha256-0GhDXGyVoMwz+vVFdplv7ObTiTRTvQDd9r9e6616iTI=";
+
+  dontNpmBuild = true;
+
+  nativeBuildInputs = [
+    python3
+  ] ++ lib.optional stdenv.isDarwin cctools;
+
+  # Fixes `semantic-release --version` output
```

Comment:
It is a bit funny that a package about semantic version needs a fixup on semantic version...

Reply:
Yeah, although, I think it is done this way so the tool itself takes care of updating the version during a release process